configs/components/xios/xios.yaml

```diff
@@ -55,10 +61,16 @@ destination: xios
 git-repository: https://git.geomar.de/foci/src/xios.git
 install_bins: bin/xios.x
 use_oasis: ''
-# 
+
+compiletime_environment_changes:
+  choose_computer.name:
```

Comment:
Why do you need `ncl` during compile time? I know it doesn't hurt but I'm surprised to see it here.

Reply:
I actually don't know but it might have to do with some of the libraries used for AWICM3, compiled by Jan. XIOS does not compile without it. @JanStreffing might be able to add more information, tbh I have not look in detail into it, but I agree, that should be removed in some point.

Reply:
I don't know about xios/ncl, but I can tell why it's in awicm3. I think is actually not needed at compile time. I was loading it in the ksh tools for some use during post-processing.

Reply:
Ok, so I will remove it with my next commit (I have some small changes to xios.yaml to be pushed anyhow)